.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-
+dist: xenial
```

Comment:
wait why? it isn't necessary I think

Reply:
The documentation [here](https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-to-run-tests-that-require-a-gui) helped me learn that xfvb is now to be added with services rather than by the previous command (which was failing). It said "this will start xvfb and set the right values for the DISPLAY environment variable". Shall I remove dist: xenial?